portal/v2/src/ClusterDetailListComponents/ClusterOperatorsWrapper.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState, useEffect, useRef } from "react"
+import { AxiosResponse } from 'axios';
+import { FetchClusterOperators } from '../Request';
+import { ICluster } from "../App"
+import { IMessageBarStyles, MessageBar, MessageBarType, Stack } from '@fluentui/react';
+import { clusterOperatorsKey } from "../ClusterDetail";
+import { ClusterOperatorListComponent } from "./ClusterOperatorList";
+
+export interface ICondition {
+  status: string,
+  reason: string,
+  lastUpdated: string,
+  message: string
```

Comment:
should we also add `type: string` here as well?

Reply:
So we don't need to actually add it here as the type is what is used for the header of the InfoList
![image](https://user-images.githubusercontent.com/81722599/207469366-58acd05a-6876-4982-884e-357a6c88bda5.png)

If we added type to this interface as well the heading would double up with an entry to the InfoList